modules/monitoring-searching-and-filtering-metrics-targets.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * monitoring/managing-metrics-targets.adoc
+
+[id="monitoring-searching-and-filtering-metrics-targets_{context}"]
+= Searching and filtering metrics targets
+
+The list of metrics targets can be long. You can filter and search these targets based on various criteria.
+
+In the *Administrator* perspective, the *Metrics Targets* page provides details about targets for default {product-title} and user-defined projects.
+This page lists the following information for each target:
+
+* the service endpoint URL being scraped
+* the ServiceMonitor component being monitored
```

Comment:
I assume that PodMonitor would also work? @kyoto for confirmation.

Reply:
No, this is currently just displaying ServiceMonitors.